Lib/fontmake/font_project.py

```diff
@@ -25,19 +25,12 @@
 from cu2qu.rf import fonts_to_quadratic
 from fontTools import subset
 from glyphs2ufo.glyphslib import build_masters, build_instances
+from mutatorMath.ufo import build as build_designspace
+from robofab.world import OpenFont
```

Comment:
hmm. why do you [drop the robofab dependency in ufoLib](https://github.com/googlei18n/ufo2ft/pull/31) while at the same time you require it in fontmake?